pymilvus/orm/iterator.py

```diff
@@ -192,31 +195,18 @@ def close(self) -> None:
 
 
 def metrics_positive_related(metrics: str) -> bool:
-    if metrics is CALC_DIST_L2:
+    if (
+        metrics is CALC_DIST_L2
```

Comment:
how about 
` return metrics in [CALC_DIST_L2, CALC_DIST_JACCARD, CALC_DIST_HAMMING, CALC_DIST_TANIMOTO]`

Reply:
fine, changed. I just know there is 'in' keyword in python